### PR TITLE
fix(config): add unit and mention range for Qubino ZMNHTD1 param 42

### PR DESCRIPTION
**Pull request by fojie**
PR description here
Fix error in param #42 for ZMNHTD1. Adjusted according to Qubino User manual.

### CODE REVIEW

File: packages/config/config/devices/0x0159/zmnhtd.json [hunk 1]
```diff
@@ -98,12 +98,12 @@
 		},
 		{
 			"#": "42",
-			"label": "Power reporting (Watts) by time interval",
-			"unit": "%",
+			"label": "Power reporting (Watts) by time interval (0-59 = reporting disabled)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0159/zmnhtd.json:101]:
```suggestion
			"label": "Power (W) Reporting Interval",
			"description": "Allowable range: 60-32535",
```
and under the default value add an option:
```jsonc
			"options": [
				{
					"label": "Disable",
					"value": 0
				}
			]
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]

**Comment by AlCalzone** [comment]
@fojie are you planning to finish this?

**Comment by fojie** [comment]
Hi,

Sorry for this. This is my first ever PR, and to be honest I don't have time to learn how to do the steps remaining. I thought the first step I did was enough, but obviously the PR could be improved and I don't know how to go further.

So, I suggest that I stick to just finding bugs, and let others – that handles PRs in a proper way –  do the PRs.

Is this OK? I'll gladly hand it over to you. Is that's possible I think it takes less time for you, than to try to tech my how to do it.

	Best Regards
	Folke

**Comment by AlCalzone** [comment]
Its not that hard, I described the two necessary changes above - you'll just need to edit the file again and do the changes. Its okay though if you don't feel confident here.

**Pull request closed without merging**